functions/Copy-DbaDatabase.ps1

```diff
@@ -1013,7 +1013,7 @@ function Copy-DbaDatabase {
 						$dropResult = Remove-SqlDatabase $destServer $dbName
 						
 						if ($dropResult -eq $false) {
-							Write-Message -Level Verbose -Message "Database could not be dropped. Aborting routine for this database"
+							Write-Message -Level Verbose -Message "Database could not be dropped. Aborting routine for this database'"
```

Comment:
This should have `this database."` instead of `this database'"`

Reply:
Yes, that should be a period. Sorry